src/build.py

```diff
@@ -840,6 +858,27 @@ def V8():
     CopyBinaryToArchive(os.path.join(V8_OUT_DIR, a))
 
 
+def Jsc():
+  buildbot.Step('JSC')
+  Mkdir(JSC_OUT_DIR)
+
+  proc.check_call(['xcrun', PREBUILT_CMAKE_BIN, '-Wno-dev',
+                   '..', '-G', 'Ninja',
+                   '-DCMAKE_BUILD_TYPE="Release"',
+                   '-DPORT=Mac',
+                   '-DENABLE_WEBASSEMBLY=ON'],
+                  cwd=JSC_OUT_DIR)
+  proc.check_call(['ninja', 'jsc'], cwd=JSC_OUT_DIR)
+  proc.check_call(['../Tools/Scripts/run-javascriptcore-tests',
+                   '--root=bin',
+                   '--filter', 'wasm',
+                   '--no-build', '--no-testapi'],
+                  cwd=JSC_OUT_DIR)
+  to_archive = [Executable(os.path.join('bin', 'jsc'))]
+  for a in to_archive:
+    CopyBinaryToArchive(os.path.join(JSC_OUT_DIR, a))
```

Comment:
any idea approximately how big this output is? Is it just the jsc binary, or does it need supporting files?